service/controller/azureconfig/handler/masters/create_master_instances_upgrading.go

```diff
@@ -128,6 +138,27 @@ func (r *Resource) masterInstancesUpgradingTransition(ctx context.Context, obj i
 	return currentState, nil
 }
 
+// isMastersVmssUpToDate checks whether or not the masters VMSS has been updated. We rely on the tag containing the az-op
+// version, as we are mostly interested on the cloudconfig blob, which depends on the az-op version.
+func (r *Resource) isMastersVmssUpToDate(ctx context.Context, azureConfig *providerv1alpha1.AzureConfig) (bool, error) {
+	virtualMachineScaleSetsClient, err := r.ClientFactory.GetVirtualMachineScaleSetsClient(ctx, azureConfig.ObjectMeta)
+	if err != nil {
+		return false, microerror.Mask(err)
+	}
+
+	mastersVMSS, err := virtualMachineScaleSetsClient.Get(ctx, key.ClusterID(azureConfig), key.MasterVMSSName(*azureConfig))
+	if err != nil {
+		return false, microerror.Mask(err)
+	}
+
+	azureOperatorVersionTag, ok := mastersVMSS.Tags["gs-azure-operator.giantswarm.io-version"]
```

Comment:
Don't we have the label in a const anywhere?

Reply:
I looked for it but we haven't because we use it directly in the json template, not in Go code.

Reply:
alright, LGTM